core/arch/arm/plat-hikey/platform_config.h

```diff
@@ -38,10 +38,10 @@
 #endif /* ARM64 */
 
 /* PL011 UART */
-#if defined(CFG_HIKEY_UART3)
-#define CONSOLE_UART_BASE	0xF7113000
-#else
+#if defined(CFG_HIKEY_UART0)
```

Comment:
Exactly, although some other projects may not use the `CFG_` prefix so it may be better to stick to their own convention if any ;-)